tests/unit/experimental/components/test_entity_relation_extractor.py

```diff
@@ -154,8 +155,8 @@ async def test_extractor_llm_badly_formatted_json() -> None:
         llm=llm,
     )
     chunks = TextChunks(chunks=[TextChunk(text="some text", index=0)])
-    with pytest.raises(LLMGenerationError):
-        await extractor.run(chunks=chunks)
+
+    await extractor.run(chunks=chunks)
```

Comment:
Do we need to add some `assert` here? Maybe to check the json was actually fixed as we expect? (or maybe remove this test if it's the same as `test_extractor_llm_badly_formatted_json_gets_fixed`?)

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/226/commits/ab42906ebdf1d9d63682081132d62b1a9e1fb419

I've modified the test scenario to have an unfixable json